src/sentry/static/sentry/app/components/acl/feature.tsx

```diff
@@ -193,22 +190,4 @@ class Feature extends React.Component<FeatureProps> {
   }
 }
 
-type FeatureContainerProps = BaseFeatureProps & {
-  config: {[key: string]: string};
-};
-
-class FeatureContainer extends React.Component<FeatureContainerProps> {
-  static propTypes = {
-    config: SentryTypes.Config.isRequired,
-  };
-
-  render() {
-    const features = this.props.config.features
-      ? Array.from(this.props.config.features)
-      : [];
-
-    return <Feature configFeatures={features} {...this.props} />;
-  }
-}
-
-export default withConfig(withOrganization(withProject(FeatureContainer)));
+export default withOrganization(withProject(withConfig(Feature)));
```

Comment:
@EvanPurkhiser what's the reason for switching the order of the HoCs?

Reply:
oops. This was me testing something actually, looks like I didn't put it back. Though in this case order isn't really important.

Reply:
I can undo, but I don't really want to wait for tests again and it shouldn't affect anything

Reply:
@EvanPurkhiser No worries, I was just curious